### ISSUE TITLE
Track modification times for ClassSubject

### ISSUE THREAD

**Issue by jmoldow** [learning-unlimited/ESP-Website#1372]
This can be useful in various situations. For example, right now I am running some queries on Stanford classes, but I will need to do this again later for the classes that are registered after I do this. It would be useful to be able to only process classes that were created after I did my first round of queries.

The modification time should be affected not just by editing the class details, but also by adding / removing co-teachers.

**Comment by willgearty** [comment]
Any progress on this? I'd like to work on a teacher big board at some point and we'll need this first.

**Comment by benjaminjkraft** [comment]
Creation times were added in #2013.  (They'll only exist for classes after that was merged but that should be good enough; any backfilling would have to be on a chapter-by-chapter basis since we'd be getting the data from emails and would be Just For Funsies.)

**Comment by Dhirenderchoudhary** [comment]
@willgearty if this issue is not resolved please assign it to me.

**Comment by willgearty** [comment]
I'm not sure how useful this would be. @hwatheod @milescalabresi what do you think?

**Comment by hwatheod** [comment]
Yeah, I'm also not sure how useful this would be. From the description, it sounds like @jmoldow was manually running queries (from the Django shell, I guess), which is something most admins will not be able to do. If something requires frequent manual queries, it should probably be added as a feature to the website.

**Comment by Dhirenderchoudhary** [comment]
If we have to add feature or anything related i'll love to do that please just let me know.

**Comment by willgearty** [comment]
I feel like an easy way to achieve this would be through django-reversion? But I don't know what migrating an existing model to django-reversion looks like.

**Comment by Gourigupta2004** [comment]
Hi @willgearty, I'm interested in working on this issue. I'm a backend developer and have experience with Django. I'd like to take this up — can I be assigned?
